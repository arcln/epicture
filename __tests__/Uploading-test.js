import 'react-native';
import React from 'react';
import Uploading from '../screens/UploadingScreen';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';

describe('Uploading snapshot', () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it('renders the screen', async () => {
    const tree = renderer.create(<Uploading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
