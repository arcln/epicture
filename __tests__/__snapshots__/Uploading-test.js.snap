// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Uploading snapshot renders the screen 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <ExponentLinearGradient
    colors={
      Array [
        4278190080,
        4278190080,
      ]
    }
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "height": 300,
        }
      }
    >
      <Image
        onError={[Function]}
        onHeightChange={[Function]}
        source={1}
        style={
          Array [
            Object {
              "height": 0,
              "width": 300,
            },
            undefined,
          ]
        }
      />
      <Text
        style={
          Object {
            "color": "#fff",
            "marginTop": 100,
          }
        }
      >
        Your image is being processed...
      </Text>
    </View>
  </ExponentLinearGradient>
</View>
`;
